ftests/046-cgexec-empty_controller.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python3
+#
+# cgexec to an empty cgroup functionality test using the python bindings
+#
+# Copyright (c) 2022 Oracle and/or its affiliates.
+# Author: Kamalesh Babulal <kamalesh.babulal@oracle.com>
+#
+
+from cgroup import Cgroup, CgroupVersion
+from run import Run
+import consts
+import ftests
+import sys
+import os
+
+CONTROLLER = 'cpu'
+CGNAME = '046cgexec'
+
+
+def prereqs(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    if CgroupVersion.get_version(CONTROLLER) != CgroupVersion.CGROUP_V2:
+        result = consts.TEST_SKIPPED
+        cause = 'This test requires cgroup v2'
+
+    return result, cause
+
+
+def setup(config):
+    Cgroup.create(config, None, cgname=CGNAME)
+
+
+def test(config):
+    result = consts.TEST_PASSED
+    cause = None
+
+    config.process.create_process_in_cgroup(config, '', CGNAME)
+    output = Cgroup.get(
+                         config, controller=None, cgname=CGNAME,
+                         setting='cgroup.procs', print_headers=True,
```

Comment:
`print_header=True` will cause there to always be some text in `output`.  This means that the `len(output)` check will never fail.

Reply:
I changed it to False.  Let me know if you have any concerns with this change.

Reply:
@drakenclimber Thanks for the review and fixing the `print_headers`.